kolibri/plugins/learn/kolibri_plugin.py

```diff
@@ -12,10 +10,6 @@ class LearnModule(KolibriFrontEndPluginBase):
     """
     entry_file = "assets/src/learn.js"
 
-    events = {
```

Comment:
Removing this event mapping means that the plugin will not actually do anything currently, unless you tell it to do so during initialization.